cmd/flag/proxy_flags.go

```diff
@@ -30,38 +30,50 @@ var (
 	defaultProxyProtocol = "udp"
 
 	// Supported proxy.
-	availableProxy = []string{"udp"}
-	// Default listen host
-	defaultListenHost = "127.0.0.1"
-	// Default listen port
-	defaultListenPort uint16 = 6000
-	// Default send host
-	defaultSendHost = "127.0.0.1"
-	// Default send port
-	defaultSendPort uint16 = 6001
+	availableProxy = []string{"udp", "tcp"}
+	// Default listen host for UDP.
+	defaultUDPListenHost = "127.0.0.1"
+	// Default listen port for UDP.
+	defaultUDPListenPort uint16 = 6000
+	// Default send host for UDP.
+	defaultUDPSendHost = "127.0.0.1"
+	// Default send port for UDP.
+	defaultUDPSendPort uint16 = 6001
+
+	// Default listen host for TCP.
+	defaultTCPListenHost = "127.0.0.1"
+	// Default listen port for TCP.
+	defaultTCPListenPort uint16 = 4000
```

Comment:
Think this can be 6000, don't see a reason to use different default port for TCP and UDP